### PR TITLE
Add Health Analyzer to Medical Intern PDA

### PR DESCRIPTION
**Pull request by LordEclipse**
## About the PR 
When I played Medical Intern for those sweet role timer points, I noticed that their PDA doesn't have a health analyzer, and it felt a little weird, so I changed it.

**Changelog**

:cl:
- tweak: Medical Intern PDA's now have Health Analyzers built into them.

### CODE REVIEW

**Comment by LordEclipse** [comment]
So the checks failed, what does that mean

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml [hunk 1]
```diff
@@ -116,6 +116,8 @@
       state: pda-internmed
   - type: Icon
     state: pda-internmed
+    - type: HealthAnalyzer
```
Comment (Just-a-Unity-Dev) [inline Resources/Prototypes/Entities/Objects/Devices/pda.yml:119]: You didn't indent this properly, follow 2 spaces for YAML.

**Review by Just-a-Unity-Dev — CHANGES_REQUESTED** [review]
Everything else LGTM, just a bit of misindented YAML.

**Comment by asperger-sind** [comment]
> So the checks failed, what does that mean

Means code is shitfarting itself and doesn't work properly

**Comment by Just-a-Unity-Dev** [comment]
> > So the checks failed, what does that mean
> 
> Means code is shitfarting itself and doesn't work properly

Reeaaall helpful. Could've at least told click [Details](https://github.com/space-wizards/space-station-14/runs/7964935931?check_suite_focus=true) on his checks.

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml (hunk 1 above)
Comment (LordEclipse) [reply Resources/Prototypes/Entities/Objects/Devices/pda.yml]: That SHOULD fix it, hopefully.

**Review by Just-a-Unity-Dev — CHANGES_REQUESTED** [review]
Looks good to me

**Comment by Just-a-Unity-Dev** [comment]
wtf i was supposed to approve that not request changes

**Review by Just-a-Unity-Dev — APPROVED** [review]
good pr

**Pull request merged**